src/main/java/io/vertx/ext/sql/SQLConnection.java

```diff
@@ -34,6 +35,18 @@
 public interface SQLConnection extends AutoCloseable {
 
   /**
+   * Sets the desired options to be applied to the connection when a statement is executed.
+   *
+   * The options should be applied on a connection basis and are not applied to new connections automatically.
```

Comment:
At the risk of sounding picky, this comment is not clear to me. Will I have to apply these options any time I acquire a connection from the pool?